src/plugin.js

```diff
@@ -426,18 +426,18 @@ class vttThumbnailsPlugin {
   }
 
   /**
-   * getSecondsFromTimestamp
+   * getMillisecondsFromTimestamp
    *
    * @param  {string} timestamp VTT timestamp
-   * @return {number}           timestamp in seconds
+   * @return {number}           timestamp in milliseconds
    */
-  getSecondsFromTimestamp(timestamp) {
+  getMillisecondsFromTimestamp(timestamp) {
```

Comment:
this method still return seconds and not milliseconds

Reply:
Yeah I agree and there's a typo lol. Not sure what's going on here lol